metaflow/plugins/list_cli.py

```diff
@@ -0,0 +1,101 @@
+from metaflow._vendor import click
+from metaflow import Flow, Run
+from metaflow import util
+from metaflow.exception import MetaflowNotFound, CommandException
+import json
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group(help="List objects pertaining to your flow.")
+def list():
+    pass
+
+
+def _fetch_runs(flow_name, num_runs, namespace):
+    counter = 1
+    try:
+        flow = Flow(flow_name)
+    except MetaflowNotFound:
+        flow = None
+    run_list = []
+    if flow:
+        for run in Flow(flow_name).runs():
+            if counter > num_runs:
+                break
+            tags = [t for t in run.tags]
```

Comment:
you don't need this. the call to `namespace(namespace)` should take care of filtering for you (it was already filtered by username by default).

Reply:
Gotcha, ok I changed it to setting the namespace before retrieving runs, instead of manually filtering tags [as described here](https://github.com/Netflix/metaflow/pull/1037#issuecomment-1123825173)